main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/fsouza/fake-gcs-server/fakestorage"
+)
+
+func generateObjectsFromFiles() []fakestorage.Object {
+	objects := []fakestorage.Object{}
+
+	// if a /data volume is mounted in the container
+	if _, err := os.Stat("/data"); !os.IsNotExist(err) {
+		// list the content
+		files, err := ioutil.ReadDir("/data")
+
+		if err != nil {
+			panic(err)
+		}
+
+		for _, f := range files {
+			fn := f.Name()
+			fi, err := os.Stat("/data/" + fn)
+
+			if err != nil {
+				panic(err)
+			}
+
+			switch mode := fi.Mode(); {
```

Comment:
I think we should make ``gocritic`` happy and switch this to an ``if`` statement (see [TravisCI output](https://travis-ci.com/fsouza/fake-gcs-server/jobs/216685618#L327-L330)).